web/src/components/molecules/Asset/Viewers/MvtViewer/Imagery.tsx

```
@@ -1,19 +1,28 @@
 import styled from "@emotion/styled";
 import { VectorTileFeature } from "@mapbox/vector-tile";
-import { Cartesian3, Math, BoundingSphere, HeadingPitchRange } from "cesium";
+import {
+  Cartesian3,
+  Math,
```

Biome (1.9.4)

[error] Do not shadow the global "Math" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)